### PR TITLE
Multistage for build

### PR DESCRIPTION
**Pull request by fcrisciani**
Signed-off-by: Flavio Crisciani <flavio.crisciani@docker.com>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ jobs:
     steps:
       - checkout
       - setup_remote_docker:
+          version: 17.11.0-ce
```
Comment (thaJeztah) [inline .circleci/config.yml]: This is the latest version they support? 😞

Comment (fcrisciani) [reply .circleci/config.yml]: from the doc looks like is the most recent, is under "edge" :D
Did not try to configure above the doc, maybe is just outdated..

Comment (thaJeztah) [reply .circleci/config.yml]: I'm sure @dnephin knows 😅

Comment (fcrisciani) [reply .circleci/config.yml]: tried actually the latest 18.05 and seems to be scheduled the job

**Comment by selansen** [comment]
I started build locally with your changes on my mac. Not sure if it is just temp issue or real issue.

--- FAIL: TestSandboxCreate (0.10s)
	sandbox_test.go:60: Failed to set ipv6 gateway to sandbox: route for the gateway fe80::1 could not be found: network is unreachable

**Comment by selansen** [comment]
LGTM

**Pull request merged**